### DISCUSSION TITLE
Shawn's Blog

### DISCUSSION THREAD

**Discussion by ShawnNotFound** [MaLuns/hexo-theme-async#81] [Show 分享展示]
Name: Shawn's Blog
Url: http://blog.shawn404.top/
Avatar: https://tse4-mm.cn.bing.net/th/id/OIP-C.xNLWrybyA5rgSUMPR9AozgAAAA?pid=ImgDet&rs=1
Desc: Strive hard and work novelty

求一个展示位Orz

**Comment by MaLuns** [comment]
已加
